### ISSUE TITLE
Breaks when used with inlining tools

### ISSUE THREAD

**Issue by paulvarache** [Juicy/juicy-ace-editor#13]
Using this component with an inlining tool like [vulcanize](https://github.com/Polymer/vulcanize), breaks it.
This is due to the fact that the lookup for the template is done using this code:

```js
// Refers to the "importer", which is index.html
var thatDoc = document;
// Refers to the "importee", which is juicy-ace-editor.html
var thisDoc = (document._currentScript || document.currentScript).ownerDocument;
// Gets content from <template>
var template = thisDoc.querySelector('template').content;
```

This works perfectly fine when the file is imported by the browser but fails when inlined cause thisDoc refers now to the whole document and template ends up being the first template in the page, not necessarily the components' template.

**Comment by tomalec** [comment]
Thanks for reporting and fixing it :)

**Comment by paulvarache** [comment]
Fixed in #14
